packages/coding-standards/src/Phpstan/OrmPropertyGetterHasNoTypehintRule.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\Phpstan;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MissingPropertyFromReflectionException;
+use PHPStan\Reflection\PropertyReflection;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+
+class OrmPropertyGetterHasNoTypehintRule implements Rule
+{
+    private const CHECKED_NAMESPACE = 'Shopsys\\';
+
+    /**
+     * @return string
+     */
+    public function getNodeType(): string
+    {
+        return ClassMethod::class;
+    }
+
+    /**
+     * @param \PhpParser\Node $node
+     * @param \PHPStan\Analyser\Scope $scope
+     * @return array
+     */
+    public function processNode(Node $node, Scope $scope): array
+    {
+        if (!$node instanceof ClassMethod) {
+            return [];
+        }
+
+        if (!str_starts_with($scope->getClassReflection()?->getName(), self::CHECKED_NAMESPACE)) {
+            return [];
+        }
+
+        if ($node->getReturnType() === null) {
+            return [];
+        }
+
+        $methodName = $node->name->toString();
+
+        if (!str_starts_with($methodName, 'get')) {
```

Comment:
What about setters? I think that same rule applies there.

Reply:
added setter rule and applied